pytorch_lightning/trainer/connectors/logger_connector/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pytorch_lightning.trainer.connectors.logger_connector.epoch_loop_result import EpochResultStore, LoggerStages
+from pytorch_lightning.trainer.connectors.logger_connector.callback_hook_validator import CallbackHookNameValidator
```

Comment:
I think, we shouldn't export these. They're only utils for `LoggerConnector`, right? So every interaction with them is  supposed to be internal?

Reply:
Yes.